src/app/plugins/broadcast.plugin.ts

```diff
@@ -0,0 +1,71 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType, ClassType, Maybe } from '../../common/types';
+import Constants from '../../common/constants';
+import { RichEmbed, TextChannel } from 'discord.js';
+
+export class BroadcastPlugin extends Plugin {
+  public name: string = 'Broadcast';
+  public description: string = 'Sends an annoucement to all class channels';
+  public usage: string = '!broadcast <message>';
+  public pluginAlias = [];
+  public permission: ChannelType = ChannelType.Admin;
+  public pluginChannelName: string = Constants.Channels.Staff.AdminChat;
+
+  private _LION_PFP_URL: string =
+    'https://cdn.discordapp.com/avatars/574623716638720000/7d404c72a6fccb4a3bc610490f8d7b72.png';
```

Comment:
is it worth making this a global constant

Reply:
Probably

Reply:
Should I put it in Constants.ts

Reply:
yes